web-frontend/cypress/e2e/acceptance/home/send.cy.tsx

```diff
@@ -54,47 +113,197 @@ describe('E2E | Acceptance | Home', () => {
+        .click();
+      cy.url().should('eq', `${baseUrl}/${account.nickname}/home`);
+    });
+
+    it('should process any % as input', () => {
+      const account = mockedAccounts.at(1);
+      const recipientAccount = mockedAccounts.at(2);
+      const standardFees = '1000';
+
+      navigateToTransfercoins(1);
+
+      // we have to wait or else balance is not initialized properly
+      cy.wait(1000);
+
+      performPercentTest(0.25, account, standardFees);
+      performPercentTest(0.5, account, standardFees);
+      performPercentTest(0.75, account, standardFees);
+      performPercentTest(1, account, standardFees);
+    });
+
+    it('should transfer to accounts', () => {
+      const randomIndex = Math.floor(Math.random() * mockedAccounts.length);
+      const selectedAccount = mockedAccounts.at(randomIndex);
+
+      navigateToTransfercoins(2);
+
+      cy.get('[data-testid="transfer-between-accounts"]').click();
+
+      cy.get('[data-testid="popup-modal-content"').should('be.visible');
+
+      for (let i = 0; i < mockedAccounts.length; i++) {
+        cy.get(`[data-testid="selector-account-${i}"]`).should('be.visible');
+      }
+
+      cy.get(`[data-testid="selector-account-${randomIndex}"]`).click();
+      cy.get(`[data-testid="input-field"]`).should(
+        'have.value',
+        selectedAccount.address,
+      );
+    });
+
+    it('should refuse wrong currency input', () => {
+      const account = mockedAccounts.at(2);
+      const recipientAccount = mockedAccounts.at(1);
+
+      const amount = 1000.12311132;
+      const invalidAmount = 'things';
+      const tooMuch = Number(account?.candidateBalance) + 1000;
+      const tooLow = 0;
+      const notEnoughForFees = Number(account?.candidateBalance);
+
+      const standardFees = '1000';
+      navigateToTransfercoins(2);
+
+      cy.get('[data-testid="currency-field"')
+        .should('exist')
+        .type(invalidAmount);
+
+      cy.get('[data-testid="input-field"').type(recipientAccount.address);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
+
+      cy.get('[data-testid="input-field-message"]').should(
+        'contain.text',
+        'Invalid amount',
+      );
+
+      cy.get('[data-testid="currency-field"').clear().type(tooMuch);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
+
+      cy.get('[data-testid="input-field-message"]').should(
+        'contain.text',
+        'Insufficient funds',
+      );
+
+      cy.get('[data-testid="currency-field"').clear().type(tooLow);
+
+      cy.get('[data-testid="button"]').contains('Send').click();
+
+      cy.get('[data-testid="input-field-message"]').should(
+        'contain.text',
+        'Amount must be greater than zero',
+      );
+
+      // testing amount + fees to hugh seems tricky
+      // amount + fees to high also triggers insufficient funds
+      // there mayeb we can refacto it ?
+      // See sendForm.tsx line 84
```

Comment:
Tests should explain it self. If we are doing extra comments we may rethink the test or fix what is not clear...